get-users/creative-guidelines.md

```diff
@@ -0,0 +1,40 @@
+# Creative Guidelines 
+
+
+## Ad Units
+
+Promotional creatives can be banners or full screen interstitials. To learn more about what types of ads we support, and how they are displayed, check out our [ad units](doc/make-money/ad-overview) documentation.
+
+## Performance
+
+You want to optimize your creatives for top performance. If you achieve a higher TTR (tap through rate) and CR (conversion rate) your creative will waste less valuable impressions.  This will result in better ads being displayed on your app, with you earning more tap credits. 
+
+## Generating Creatives
+
+Our creative generation system makes uploading promotional creatives easy! All you need to do is provide some basic information about your application. Our system will then generate creatives based on that information and screen shots of your application from the app store. Please,  no profanities allowed! 
+
+### Theme
+
+You have three options for your creative theme: light, dark and app icon colors.  Check out all three and choose the one that you think makes your creative look the best.
+
+### Description
+
+You have **32** characters to convey a strong message on what your app is about. Use actionable words if possible like “create ringtones” or “play now”. 
+
+### App Name
+
+Let people know the name of your app. Names are limited to **20** characters.
+
+### App Price
+
+The price of your app will automatically be placed in the creative for you.
+
+## Custom Banners
+
+If you have custom promotional creatives for your application you can upload them to use instead. Custom creatives tend to perform better as they give a better representation of what the app experience is like. A well designed cretaive can radically improve your overall performance. Just be sure to show your app’s icon somewhere on the creative.
```

Comment:
misspelling... 'a well designed _creative_ can radically...'

Also 'apps' should not have an apostrophe.